Makefile

```diff
@@ -136,6 +136,15 @@ docker-build: test ## Build docker image with the manager.
 docker-push: ## Push docker image with the manager.
 	$(CONTAINER_TOOL) push ${IMG}
 
+.PHONY: images-build
+images: test ## Build all container images
```

Comment:
I would add ``ifneq (, $(IMAGE_REGISTRY))`` to the top of the Makefile
so then you can construct the image name later on.
```
IMAGE_TAG_BASE = $(IMAGE_REGISTRY)/$(IMAGE_NAME)
IMAGE_TAG_LATEST?=$(IMAGE_TAG_BASE):latest
```

etc.